src/platform/NodePlatformFunctions.ts

```diff
@@ -119,6 +126,28 @@ class NodePlatformFunctions extends PlatformFunctions {
         });
     });
   }
+
+  /** @override */
+  createNodeHttpClient(agent: http.Agent): typeof HttpClient {
+    // @ts-ignore
```

Comment:
why ignore?

Reply:
I get a this error:
```
Type 'NodeHttpClient' is missing the following properties from type 'typeof HttpClient': prototype, CONNECTION_CLOSED_ERROR_CODES, TIMEOUT_ERROR_CODE, makeTimeoutError
```
`CONNECTION_CLOSED_ERROR_CODES`, `TIMEOUT_ERROR_CODE`, and `makeTimeoutError` are defined as static properties / methods on `HttpClient`, but I tried and failed at diagnosing why they were't being detected by the linter.

Reply:
Since this is due to how we import `HttpClient` and `NodeHttpClient` and can be fixed with ES modules, I'm going to wait to fix this when we do the ESM refactor.